### PR TITLE
[HOTFIX] handling serialization/desrialization of the files properly;

### PR DESCRIPTION
**Pull request by opalczynski**

### CODE REVIEW

**Comment by opalczynski** [comment]
Note to myself: add extended tests with files in Object.

**Comment by 23doors** [comment]
4 correct isort commits where only one deals with imports? ;)

**Comment by opalczynski** [comment]
yep :) my bad - hope it will be fast fix :P

**Comment by opalczynski** [comment]
@23doors @Zhebr Can you take a look?

File: tests/integration_test_data_objects.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+from hashlib import md5
+
+import requests
+import six
+from syncano.models import Object
+from tests.integration_test import InstanceMixin, IntegrationTest
+
+try:
+    # python2
+    from StringIO import StringIO
```
Comment (23doors) [inline tests/integration_test_data_objects.py]: cStringIO is a drop-in replacement which is much more performant

**Comment by opalczynski** [comment]
@23doors I promise improvement ;)

**Pull request closed without merging**